invenio_records_marc21/ui/theme/assets/semantic-ui/translations/invenio_records_marc21/messages/en/translations.json

```diff
@@ -10,10 +10,7 @@
   "Manage": "Manage",
   "Version": "Version",
   "Preview": "Preview",
-  "{{count}} results found": "{{count}} results found",
```

Comment:
Are they translations replaced by other strings?

Reply:
This translation is no longer in use since we are utilizing the count component from the search application.

Reply:
so we use the translation strings from them?

Reply:
Yes